modules/naveggId.js

```diff
@@ -0,0 +1,83 @@
+/**
+ * This module adds naveggId to the User ID module
+ * The {@link module:modules/userId} module is required
+ * @module modules/naveggId
+ * @requires module:modules/userId
+ */
+import * as utils from '../src/utils.js'
+import { submodule } from '../src/hook.js';
+import { getStorageManager } from '../src/storageManager.js';
+
+const MODULE_NAME = 'naveggId';
+export const storage = getStorageManager();
+
+function readnaveggIdFromLocalStorage() {
+  return storage.localStorageIsEnabled ? storage.getDataFromLocalStorage('nvggid') : null;
+}
+
+function readnaveggIDFromCookie() {
+  return storage.cookiesAreEnabled ? storage.getCookie('nvggid') : null;
+}
+
+function readoldnaveggIDFromCookie() {
+  return storage.cookiesAreEnabled ? storage.getCookie('nid') : null;
```

Comment:
would be easier to read if you stored const variables and referenced them here: const OLD_NAVEGG_ID = 'nid' and const NAVEGG_ID = 'nvggid'